config/src/config/mod.rs

```diff
@@ -398,15 +394,33 @@ mod test {
 
     #[test]
     fn verify_all_configs() {
+        // First test well defined configs
         let _ = vec![
-            PathBuf::from("data/configs/overrides/persistent_data.node.config.override.toml"),
-            PathBuf::from("data/configs/single.node.config.toml"),
-            PathBuf::from("../terraform/validator-sets/100/fn/node.config.toml"),
-            PathBuf::from("../terraform/validator-sets/100/val/node.config.toml"),
-            PathBuf::from("../terraform/validator-sets/dev/fn/node.config.toml"),
-            PathBuf::from("../terraform/validator-sets/dev/val/node.config.toml"),
+            "data/configs/single.node.config.toml",
         ]
         .iter()
-        .map(|path| NodeConfig::load(path).expect("NodeConfig"));
+        .map(|path| NodeConfig::load(PathBuf::from(path)).expect(&format!("Error in {}", path)))
+        .collect::<Vec<_>>();
```

Comment:
Sorry i missed this in the original series.

Reply:
Hahaha, in Hack there was a performance reason to use maps in that you could more easily parallelize your async / awaits... Now I default to it like a functional programmer.